test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Encoders/LengthPrefixedTextMessageFormatterTests.cs

```diff
@@ -1,46 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.IO;
-using System.Text;
-using Microsoft.AspNetCore.SignalR.Internal.Encoders;
-using Xunit;
-
-namespace Microsoft.AspNetCore.SignalR.Tests.Internal.Encoders
-{
-    public class LengthPrefixedTextMessageFormatterTests
-    {
-        [Fact]
-        public void WriteMultipleMessages()
```

Comment:
Do we have a test like this somewhere else?

Reply:
I don't think so because the test is trying to test a scenario that is/was not a thing. Before we would create a temporary stream for each message just to serialize to get the byte[]. We didn't serialize multiple messages to a stream. Now even the temporary stream is gone. I think this test was written when we had framing at the Sockets level where we would use this format for text frames. At that time it was possible to write multiple messages to a stream. The `LengthPrefixedTextMessageWriter` was then re-purposed to be used only for base64 encoding but the test was not deleted at that time.